audit-triplestore/src/test/java/org/fcrepo/camel/audit/triplestore/RouteTest.java

```diff
@@ -59,7 +59,7 @@ public void testEventProcessor() throws Exception {
         final Map<String, Object> headers = new HashMap<>();
         headers.put(JmsHeaders.BASE_URL, "http://localhost/rest");
         headers.put(JmsHeaders.IDENTIFIER, "/foo");
-        headers.put(JmsHeaders.TIMESTAMP, "1428360320168");
+        headers.put(JmsHeaders.TIMESTAMP, 1428360320168L);
```

Comment:
I was wondering that too, but it looks like fcrepo4 is setting it as a long:

https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java#L79